functionary/prompt_template/__init__.py

```diff
@@ -16,11 +17,16 @@ def get_default_prompt_template() -> PromptTemplate:
 
 
 def get_prompt_template_by_version(version: str) -> PromptTemplate:
+    if version == "v3.llama3":
+        return Llama3TemplateV3.get_prompt_template()
+
     if version == "v2.llama3":
         return Llama3Template.get_prompt_template()
 
     if version == "v1":
         return PromptTemplateV1.get_prompt_template()
+
+    assert version == "v2"
     return PromptTemplateV2.get_prompt_template()
```

Comment:
Can we use a hash map here to return the prompt template class instead? We can raise error if `version not in hash_map`.

Reply:
yes, we can